### PR TITLE
standardnotes: fix for 3.5.11

### PR DESCRIPTION
**Pull request by svrana**
This is @tasmos rebased commit in #106521

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/editors/standardnotes/default.nix [hunk 1]
```diff
@@ -1,5 +1,5 @@
 { lib, stdenv, appimageTools, autoPatchelfHook, desktop-file-utils
-  , fetchurl, runtimeShell }:
+  , fetchurl, runtimeShell, libsecret, gtk3, gsettings-desktop-schemas }:
```
Comment (SuperSandro2000) [inline pkgs/applications/editors/standardnotes/default.nix:2]:
```suggestion
, fetchurl, runtimeShell, libsecret, gtk3, gsettings-desktop-schemas }:
```

**Comment by SuperSandro2000** [comment]
@mgregoire ping

**Review by M-Gregoire — APPROVED** [review]
LGTM, thank you :)

**Comment by svrana** [comment]
thanks for 3.5.11 @tasmo  !

**Pull request merged**